autoblocks/_impl/tracer.py

```diff
@@ -112,7 +105,7 @@ def start_span(self):
                 props["span_id"] = prev_span_id
             self.set_properties(props)
 
-    def _send_event_unsafe(
+    async def _send_event_unsafe(
```

Comment:
this was called `_send_event_unsafe` because it didn't do any error handling, and all the error handling was a level above. this allowed us to try-except the entire thing, including stuff happening before sending the request (merging properties etc)

can you update to maintain the same pattern?